client/signup/jetpack-connect/index.jsx

```diff
@@ -31,6 +33,17 @@ const MINIMUM_JETPACK_VERSION = '3.9.6';
 const JetpackConnectMain = React.createClass( {
 	displayName: 'JetpackConnectSiteURLStep',
 
+	componentDidMount() {
+		let from = 'direct';
+		if ( this.props.isInstall ) {
+			from = 'jpdotcom';
+		}
+		this.props.recordTracksEvent( 'calypso_jpc_url_view', {
+			jpc_from: from,
+			user: this.props.userId
```

Comment:
`userid` is already stored as a property of events (if the user is logged in) automatically. I don't know that we need to duplicate it?